### DISCUSSION TITLE
No syntax highlighting in Firefox codespaces

### DISCUSSION THREAD

**Discussion by watchingdogs** [community/community#129456] [Codespaces]
### Select Topic Area

Bug

### Body

I have disabled all browser extensions, made sure that in this case the Python VSCode extension was installed, and that it also recognized the code as Python. I don't have any network filtering at the moment. The only characters I have seen highlighted are `[ ] { } ( )`.

Has anyone else seen this issue? How could I debug this?

**Comment by marcgreen** [comment]
Same. Used to be able to use FF for codespaces but about 1-2 months ago something changed and (1) codespaces stopped working unless I disabled tracking protection and (2) even after doing that, no syntax highlighting.

**Reply by watchingdogs** [reply]
Exactly, disabling tracking protection allowed me too to bypass the network error or whatever came up, but I would be still looking at a white wall of code.

**Comment by AmateurECE** [comment]
I experienced this issue today, on my second use of codespaces. Syntax highlighting works on my machine in Chrome, but fails in Firefox.

**Comment by data-spice** [comment]
I've seen something similar happen before. It might be related to **syntax highlighting not working properly** due to a VS Code setting or a language mode mismatch.

Here are a few things you can try:

1. **Check the bottom-right corner** of VS Code — make sure it says **“Python.”**  
   If not, click it and set the language mode manually.

2. Go to **Settings → Files: Associations** and check if `.py` files are correctly linked to `python`.

3. Try **reinstalling or disabling/re-enabling** the Python extension.

4. Open the **Command Palette (`Ctrl+Shift+P`)** → “Developer: Reload Window” to refresh VS Code.

5. If the problem persists, try opening VS Code with all other extensions disabled using:
   ```bash
   code --disable-extensions
